docs/data-sources/gaussdb_redis_instance.md

```diff
@@ -0,0 +1,83 @@
+---
+subcategory: "GaussDB"
+---
+
+# huaweicloud\_gaussdb\_redis\_instance
+
+Use this data source to get available HuaweiCloud gaussdb redis instance.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_gaussdb_redis_instance" "this" {
```

Comment:
we use `test` as resource name in document.

Reply:
updated